app/src/newdocs/content/marketplace.mdx

```diff
@@ -1,11 +1,15 @@
 import ScrollableAnchor from 'react-scrollable-anchor'
+import { Link } from 'react-router-dom'
 
+import links from '$shared/../links'
 import docsStyles from '$newdocs/components/DocsLayout/docsLayout.pcss'
 
 <ScrollableAnchor id="introduction-marketplace"><div>
 
 ## Introduction to the Marketplace
---content--
+The [marketplace](https://streamr.com/marketplace) is a Streamr developed application that handles the buying and selling of real-time data on the Streamr platform. The marketplace is filled with both free products and products created by data producers that wish to monetise their data. The currency of the marketplace is the <Link to={links.newdocs.dataToken}>DATA token</Link>.
```

Comment:
- Use capitalized formatting for product names (Marketplace, Core, and Network)
- Minor suggested edits for clarity

```suggestion
The [Marketplace](https://streamr.com/marketplace) is an application that enables users to buy and sell access to real-time data content on the Streamr Network. The Marketplace is filled with both paid and free products, offering data producers an opportunity to either monetise their data or make it freely available to everyone. Products have time-based subscription periods, and the currency used on the Marketplace is the <Link to={links.newdocs.dataToken}>DATA token</Link>.
```

Reply:
well said 👌